internal/pipe/linkedin/linkedin.go

```diff
@@ -0,0 +1,67 @@
+package linkedin
+
+import (
+	"fmt"
+
+	"github.com/apex/log"
+	"github.com/caarlos0/env/v6"
+	"github.com/goreleaser/goreleaser/internal/pipe"
+	"github.com/goreleaser/goreleaser/internal/tmpl"
+	"github.com/goreleaser/goreleaser/pkg/context"
+)
+
+const (
+	defaultMessageTemplate = `{{ .ProjectName }} {{ .Tag }} is out! Check it out at {{ .GitURL }}/releases/tag/{{ .Tag }}`
+)
+
+type Pipe struct{}
+
+func (Pipe) String() string { return "linkedin" }
+
+type Config struct {
+	AccessToken string `env:"LINKEDIN_ACCESS_TOKEN,notEmpty"`
+}
+
+func (Pipe) Default(ctx *context.Context) error {
+	if ctx.Config.Announce.LinkedIn.MessageTemplate == "" {
+		ctx.Config.Announce.LinkedIn.MessageTemplate = defaultMessageTemplate
+	}
+
+	return nil
+}
+
+func (Pipe) Announce(ctx *context.Context) error {
+	if ctx.SkipAnnounce {
+		return pipe.ErrSkipAnnounceEnabled
+	}
+	if !ctx.Config.Announce.LinkedIn.Enabled {
+		return pipe.ErrSkipDisabledPipe
+	}
+
```

Comment:
this can now be removed in favor of implementing the `Skip(*context.Context) bool` in the `Pipe`.

```suggestion
```

Please take a look at other announcer for examples. There's also a `TestSkip` on all of them :)